client/my-sites/plugins/main.jsx

```diff
@@ -390,6 +390,25 @@ export class PluginsMain extends Component {
 		);
 	}
 
+	getNavigationItems() {
+		const { search, selectedSiteSlug } = this.props;
+		const navigationItems = [
+			{ label: this.props.translate( 'Plugins' ), href: `/plugins/${ selectedSiteSlug || '' }` },
```

Comment:
Just noting that `selectedSiteSlug || ''` is used for the all sites screens where `selectedSiteSlug is null`